packages/backend/src/persistence/getDatabase.ts

```diff
@@ -126,90 +83,20 @@ export async function getDatabase(
         getNumberOfConverations: Storage.getNumberOfConversations(prisma),
         //Storage Toggle Hide Conversation
         toggleHideConversation: Storage.toggleHideConversation(prisma),
-        //Get the user db migration status
```

Comment:
If the migration should be kept the methods related to the old storage and the migration should be kept